tsconfig.json

```diff
@@ -13,6 +13,7 @@
     "jsx": "preserve",
     "incremental": true,
     "target": "ES2023",
+    "downlevelIteration": true,
```

Comment:
I don't know if you need this if you're using a new enough target - i think webstorm/pycharm is being silly